src/Sylius/Bundle/ResourceBundle/DependencyInjection/Driver/DoctrineODMDriver.php

```diff
@@ -34,9 +35,9 @@ public function getSupportedDriver()
      */
     protected function getRepositoryDefinition(array $classes)
     {
-        $repositoryClass = 'Sylius\Bundle\ResourceBundle\Doctrine\ODM\MongoDB\DocumentRepository';
+        $repositoryClass = new Parameter('sylius.mongodb-odm.repository.class');
 
-        if (isset($classes['repository'])) {
+        if (!empty($classes['repository'])) {
```

Comment:
Why? `isset()` is enough for `null` values, and this one can't be an empty string anyway...